website/docs/extensions/testing.md

```diff
@@ -19,15 +19,33 @@ The behaviour of the `scarb test` command can be changed by developers.
 To do so, provide a script named explicitly `test` in the current workspace `Scarb.toml`.
 If such script is found, Scarb will execute it instead of running the default test runner.
 
-For example, in order to tell `scarb test` to use [Starknet Forge](https://foundry-rs.github.io/starknet-foundry) for testing in
-your project, type the following:
+Scarb can be configured to use any tool in place of the default `cairo-test`, simply by changing
+this configuration:
 
 ```toml filename="Scarb.toml"
 [scripts]
-test = "snforge"
+test = "command-to-run-tests"
 ```
 
-Do not forget to properly set up Starknet Forge in your project beforehand.
+## Using Starknet Foundry
+
+[Starknet Foundry](https://foundry-rs.github.io/starknet-foundry) is another project developed
+by [Software Mansion](https://swmansion.com/) team.
+It enables advanced testing of [Starknet](https://www.starknet.io/) contracts, including fuzz testing, forking the
+network state, setting-up a
+specific contract state in your tests, and many more.
+
+In order to tell `scarb test` to use Starknet Foundry as the test runner testing in your project, define the following
+script:
```

Comment:
```suggestion
In order to tell `scarb test` to use Starknet Foundry as the test runner testing in your project, define the following:
```

with my previous suggestion, it should be now pretty clear that this is a script and this sounds repetitive to me